Week 4/Unit 07/Assignment 1/lab/src/main/java/fpt/lab/config/AppConfig.java

```diff
@@ -0,0 +1,19 @@
+package fpt.lab.config;
+
+import org.springframework.context.annotation.Bean;
+
+import fpt.lab.controller.EmployeeWork;
+import fpt.lab.model.Employee;
+
+public class AppConfig {
```

Comment:
Why you dont put the annotation @Configuration in this class but still work, pls explain.